src/main/java/com/google/cloud/tools/internal/sdk/CloudSdk.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.internal.sdk;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Default implementation of CloudSdk interface for known install locations
+ */
+public class CloudSdk {
```

Comment:
I might be missing something, but this looks a bit overkill. Is there a reason this can't live in ProcessCaller?